### PR TITLE
Issue 2999: Document fix index.md and getting started.md

### LINKED ISSUE #2999: Document fix Overview and Getting started docs
**Problem description**
1. Editing Typos
2. Checking links
3. Checking formats
4. Checking figures if any

**Problem location**
Index.md, getting-started.md

### PR DESCRIPTION
**Pull request by Aparnarr**
**Change log description**  
1.Fixed  formatting errors
2. Links checked

**Purpose of the change**  
Fixes #2999

### CODE REVIEW

File: documentation/src/docs/index.md [hunk 1]
```diff
@@ -52,21 +52,20 @@ Pravega can be installed locally or in a distributed environment. The installati
...
 
+1. [TLS](security/pravega-security-encryption.md)
+2. [Authorization, Authentication and RBAC](security/pravega-security-authorization-authentication.md)
 
 ## Support
 
-Don’t hesitate to ask! Contact the developers and community on the [mailing lists](https://groups.google.com/forum/#!forum/pravega-users) or on [slack](https://pravega-io.slack.com/) if you need any help.
-Please open an issue in [Github
-Issues](https://github.com/pravega/pravega/issues) if you find a bug.
+Don’t hesitate to ask! Contact the developers and community on the [Mailing List](https://groups.google.com/forum/#!forum/pravega-users) or on [Slack](https://pravega-io.slack.com/) if you need any help.
```
Comment (Aparnarr) [inline documentation/src/docs/index.md]: @fpj 
The above mentioned mailing list connects to google groups,!?Is that Okay?
Would you like to provide any new mail ids??Jus an ask.

File: documentation/src/docs/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ You may obtain a copy of the License at
 -->
 # Pravega Overview
 
-Pravega is an open source storage primitive implementing **Streams** for continuous and unbounded data. A Pravega stream is a durable, elastic, append-only, unbounded sequence of bytes that has good performance and strong consistency.  
+Pravega is an open source storage primitive implementing **Streams** for continuous and unbounded data. Pravega stream is durable, elastic, append-only and unbounded sequence of bytes having good performance and strong consistency.
```
Comment (andreipaduroiu) [inline documentation/src/docs/index.md]: I would suggest "A Pravega Stream is a" (back to where it was before)

File: documentation/src/docs/getting-started.md [hunk 3]
```diff
@@ -36,31 +36,30 @@ $ tar xfvz pravega-<version>.tgz
 
 This launches all the components of Pravega on your local machine.
 
-**Note:** This is for testing/demo purposes only, *do not* use this mode of deployment 
-in Production! 
+**Note:** This is for testing/demo purposes only, **do not** use this mode of deployment
+in Production!
 
 More options to run Pravega can be found in [Running Pravega](deployment/deployment.md) guide.
```
Comment (fpj) [inline documentation/src/docs/getting-started.md]: This line is actually repeated in line 51 below. Please remove the duplication.

File: documentation/src/docs/getting-started.md [hunk 4]
```diff
@@ -36,31 +36,30 @@ $ tar xfvz pravega-<version>.tgz
...
-           
 
 ```
 $ cd pravega-<version>
 $ bin/pravega-standalone
 ```
 
-By using the above commands, Pravega should be up and running very soon.
+By using the above commands, Pravega should be up and running immediately.
```
Comment (fpj) [inline documentation/src/docs/getting-started.md]: This sentence actually makes little sense. We should say:

```
The command above runs Pravega locally for development and testing purposes. It does not persist in the storage tiers like we do with a real deployment of Pravega and as such you shouldn't expect it to recover from crashes, and further, not rely on it for production use. For production use, we strongly encourage a full deployment of Pravega.
```

File: documentation/src/docs/getting-started.md [hunk 5]
```diff
@@ -36,31 +36,30 @@ $ tar xfvz pravega-<version>.tgz
...
 
 More options and additional ways to run Pravega can be found in [Running Pravega](deployment/deployment.md).
 
-## Running a sample Pravega App
+## Running a sample Pravega Application
 
-It is very simple to run a sample pravega App. Pravega maintains a separate github repository for sample applications.  It is located at:
+It is very simple to run a sample Pravega Application. Pravega maintains a separate github repository for sample applications.  It is located at:
```
Comment (fpj) [inline documentation/src/docs/getting-started.md]: I don't really like this style "It is very simple to...". Let's just be objective and say:

```
We have developed a few samples to introduce the developer to coding with Pravega here:

[Pravega Samples](https://github.com/pravega/pravega-samples)

```

File: documentation/src/docs/getting-started.md [hunk 6]
```diff
@@ -89,7 +88,7 @@ _Example HelloWorldWriter output_
 Writing message: 'hello world' with routing-key: 'helloRoutingKey' to stream 'examples / helloStream'
 ...
 ```
-See the [README.md](https://github.com/pravega/pravega-samples/blob/v0.3.2/pravega-client-examples/README.md) file in the standalone-examples for more details
+See the [README](https://github.com/pravega/pravega-samples/blob/v0.3.2/pravega-client-examples/README.md) file in the standalone-examples for more details
     on running the HelloWorldWriter with different parameters.
 
 **Run the sample "HelloWorldReader"**
```
Comment (fpj) [inline documentation/src/docs/getting-started.md]: @RaulGracia are these instructions up to date?

File: documentation/src/docs/index.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ You may obtain a copy of the License at
 -->
 # Pravega Overview
 
-Pravega is an open source storage primitive implementing **Streams** for continuous and unbounded data. A Pravega stream is a durable, elastic, append-only, unbounded sequence of bytes that has good performance and strong consistency.  
+Pravega is an open source storage primitive implementing **Streams** for continuous and unbounded data. A Pravega stream is a durable, elastic, append-only, unbounded sequence of bytes having good performance and strong consistency.
```
Comment (fpj) [inline documentation/src/docs/index.md]: This is wrong. Pravega is not a storage primitive. Pravega is a storage system that exposes stream as a first-class primitive.

There are multiple ways of saying this, but the points that we can't confuse are that:
- Pravega is a storage system
- Stream is the main primitive

File: documentation/src/docs/getting-started.md (hunk 3 above)
Comment (Aparnarr) [reply documentation/src/docs/getting-started.md]: @fpj Accepted Suggestion

File: documentation/src/docs/getting-started.md (hunk 4 above)
Comment (Aparnarr) [reply documentation/src/docs/getting-started.md]: Updated in the document.

File: documentation/src/docs/getting-started.md (hunk 5 above)
Comment (Aparnarr) [reply documentation/src/docs/getting-started.md]: updated in the document

File: documentation/src/docs/getting-started.md (hunk 6 above)
Comment (Aparnarr) [reply documentation/src/docs/getting-started.md]: yes, updated

**Comment by Aparnarr** [comment]
@fpj all comments were accepted and updated in the doc

File: documentation/src/docs/getting-started.md (hunk 6 above)
Comment (RaulGracia) [reply documentation/src/docs/getting-started.md]: Yes, they are up to date and can be used to run the samples.

**Pull request merged**